service/mantle/shipstation/ShipStationServices.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <!--set#GetCall Service gets data from oms find#Orders service and modify the data for Orders.xml screen-->
+    <service verb="set" noun="GetCall">
```

Comment:
Please add proper naming for service, as per the functionality.
You can add something like, get#OrderInfo